controller/cmd/service-mirror/probe_worker.go

```diff
@@ -13,12 +13,14 @@ import (
 )
 
 const httpGatewayTimeoutMillis = 50000
+const requireIDHeader = "l5d-require-id"
```

Comment:
nit: It may be worth defining this somewhere that both this and `controller/tap` can use.